Extend description (#2)
KasiaFoszcz/AmorphousMetals: metale_amorficzne/description/01_general.md

## Introduction

### Analyzer for Researchers at Wroclaw University of Science and Technology

> The purpose of this study is to develop an Amorphous Metals Analyzer

Reviewer: ```suggestion
The purpose of this study is to develop an _Amorphous Metals Analyzer_
```

Author: Done

temperature are not ductile and tend to fail suddenly when loaded in tension,
which limits the material applicability in reliability-critical applications, as
the impending failure is not evident. Therefore, there is considerable interest
in producing metal matrix composites consisting of a metallic glass matrix
containing dendritic particles or fibers of a ductile crystalline metal.

Perhaps the most useful property of bulk amorphous alloys is that they are true
glasses, which means that they soften and flow upon heating. This allows for
easy processing, such as by injection molding, in much the same way as polymers.
As a result, amorphous alloys have been commercialized for use in sports
equipment, medical devices, and as cases for electronic equipment.

### Usage of amorphous metals

#### Commercial

Currently the most important application is due to the special magnetic
properties of some ferromagnetic metallic glasses. The low magnetization loss is
used in high efficiency transformers (amorphous metal transformer) at line
frequency and some higher frequency transformers. Amorphous steel is a very
brittle material which makes it difficult to punch into motor laminations. Also

> electronic article surveillance (such as theft control passive ID tags,) often

Reviewer: ```suggestion
electronic article surveillance (such as theft control passive ID tags) often
```

Author: Done

0.197 by 0.157 in) pressure sensors for automobile and other industries, and
these sensors are smaller, more sensitive, and possess greater pressure
endurance compared to conventional stainless steel made from cold working.

#### Potential

Amorphous metals exhibit unique softening behavior above their glass transition
and this softening has been increasingly explored for thermoplastic forming of
metallic glasses. Such low softening temperature allows for developing
simple methods for making composites of nanoparticles (e.g. carbon nanotubes)
and bulk metallic glasses. It has been shown that metallic glasses can be
patterned on extremely small length scales ranging from 10 nm to several
millimeters. This may solve the problems of nanoimprint lithography where
expensive nano-molds made of silicon break easily. Nano-molds made from metallic
glasses are easy to fabricate and more durable than silicon molds. The superior
electronic, thermal and mechanical properties of bulk metallic glasses compared
to polymers make them a good option for developing nanocomposites for electronic
application such as field electron emission devices.

> Ti40Cu36Pd14Zr10 is believed to be noncarcinogenic, is about three times

Reviewer: Use subscript using HTML tag `<sub>`, e.g.:

```markdown
Ti<sub>40</sub>Cu<sub>36</sub>Pd<sub>14</sub>Zr<sub>10</sub>
```

will render as:

> Ti<sub>40</sub>Cu<sub>36</sub>Pd<sub>14</sub>Zr<sub>10</sub>

Author: Done

stronger than titanium, and its elastic modulus nearly matches bones. It has a
high wear resistance and does not produce abrasion powder. The alloy does not
undergo shrinkage on solidification. A surface structure can be generated that
is biologically attachable by surface modification using laser pulses, allowing
better joining with bone.

> Mg60Zn35Ca5, rapidly cooled to achieve amorphous structure, is being

Reviewer: Same here

Author: Done

investigated at Lehigh University as a biomaterial for implantation into bones
as screws, pins, or plates, to fix fractures. Unlike traditional steel or
titanium, this material dissolves in organisms at a rate of roughly 1 millimeter
per month and is replaced with bone tissue. This speed can be adjusted by
varying the content of zinc.

Bulk metallic glasses also seem to exhibit superior properties like SAM2X5-630
which has the highest recorded elastic limit for any steel alloy, according to
the researcher, essentially it has the highest threshold limit at which a

> material can withstand an impact without deforming permanently(plasticity). The

Reviewer: ```suggestion
material can withstand an impact without deforming permanently (plasticity). The
```

Author: Done

alloy can withstand pressure and stress of up to 12.5 GPa (123,000 atm) without
undergoing any permanent deformation, this is the highest impact resistance of

> any bulk metallic glass ever recorded (as of 2016) .This makes it as an

Reviewer: ```suggestion
any bulk metallic glass ever recorded (as of 2016). This makes it as an
```

Author: Done

attractive option for Armour material and other applications which requires high

> stress tolerance.[43][44][45] Additive manufacturing One challenge when

Reviewer: Stray reference indexes.

Author: Done

synthesising a metallic glass is that the techniques often only produce very
small samples, due to the need for high cooling rates. 3D-printing methods have
been suggested as a method to create larger bulk samples. Selective laser
melting (SLM) is one example of an additive manufacturing method that has been

> used to make iron based metallic glasses.[46][47] Laser foil printing (LFP) is

Reviewer: Stray reference indexes.

Author: Done

another method where foils of the amorphous metals are stacked and welded
together, layer by layer.

---

### What is nanoindentation?

In an indentation test, a hard tip whose mechanical properties are known
(frequently made of a very hard material like diamond) is pressed into a sample
whose properties are unknown. The load placed on the indenter tip is increased
as the tip penetrates further into the specimen and soon reaches a user-defined
value. At this point, the load may be held constant for a period or removed. The

> area of the residual indentation in the sample is measured and the hardness, H
> {\displaystyle H}, is defined as the maximum load, P m a x {\displaystyle
> P_{max}}, divided by the residual indentation area, A r {\displaystyle A_{r}}:

Reviewer: ```suggestion
area of the residual indentation in the sample is measured and the hardness, $H$, is defined as the maximum load, $P_{\mathrm{max}}$ divided by the residual indentation area, $A_{\mathrm{r}}$:
```

Author: Done

with the development of machines that could record small load and displacement
with high accuracy and precision. The load displacement data can be used
to determine modulus of elasticity, hardness, yield strength, fracture
toughness, scratch hardness and wear properties.

In nanoindentation small loads and tip sizes are used, so the indentation area
may only be a few square micrometres or even nanometres. This presents problems
in determining the hardness, as the contact area is not easily found. Atomic
force microscopy or scanning electron microscopy techniques may be utilized to
image the indentation, but can be quite cumbersome. Instead, an indenter with a
geometry known to high precision (usually a Berkovich tip, which has a
three-sided pyramid geometry) is employed. During the course of the instrumented
indentation process, a record of the depth of penetration is made, and then the
area of the indent is determined using the known geometry of the indentation
tip. While indenting, various parameters such as load and depth of penetration
can be measured. A record of these values can be plotted on a graph to create a

> load-displacement curve (such as the one shown in Figure 1). These curves can be

Reviewer: Missing figure

Author: Done

> used to extract mechanical properties of the material.[3]

Reviewer: Stray reference index.

Author: Done

---

#### Materials and research methods used for this study

> - ZrCu and ZrTi alloys

Reviewer: Add explanation what ZrCu and ZrTi is (i.e., expand the element names in parentheses).

Author: Done

- nanoindentation of specifically prepared, polished metal samples in the form

>   of thin coins. Indentation was performed on each sample in a matrix of 15 x 15

Reviewer: ```suggestion
  of thin coins. Indentation was performed on each sample in a matrix of 15×15
```

(i.e., use × symbol instead of x)

Author: Done

>   or 10 x 10 points in distances ranging from 5 to 20 um, force applied ranging

Reviewer: Same here

Author: Done

8. Nishiyama, N.; Amiya, K.; Inoue, A. (March 2007). "Recent progress of bulk
metallic glasses for strain-sensing devices". Materials Science and Engineering:
A. 449–451: 79–83. doi:10.1016/j.msea.2006.02.384.
9. Saotome, Y.; Iwazaki, H. (2000). "Superplastic extrusion of microgear shaft
of 10 μm in module". Microsystem Technologies. 6 (4): 126.
doi:10.1007/s005420050180. S2CID 137549527.
10. Kumar, G.; Tang, H. X.; Schroers, J. (2009). "Nanomoulding with amorphous
metals". Nature. 457 (7231): 868–872. Bibcode:2009Natur.457..868K.
doi:10.1038/nature07718. PMID 19212407. S2CID 4337794.
11. Hojati-Talemi, Pejman (2011). "High performance bulk metallic glass/carbon
nanotube composite cathodes for electron field emission". Applied Physics
Letters. 99 (19): 194104. Bibcode:2011ApPhL..99s4104H. doi:10.1063/1.3659898.
12. Maruyama, Masaaki (June 11, 2009). "Japanese Universities Develop Ti-based
Metallic Glass for Artificial Finger Joint". Tech-on. "Fixing bones with
dissolvable glass". Institute of Physics. October 1, 2009.
13. "Engineers Develop Record-Breaking Steel". Engineering.com. Retrieved
2022-06-24.
14. "Record-breaking steel could be used for body armor, shields for

> satellites". jacobsschool.ucsd.edu. Retrieved 2022-06-24.

Reviewer: 1. Properly reformat. When wrapping lines in enumeration, wrapped lines should be indented with three spaces.
2. Use italics where applicable (similar to how it is on Wikipedia).
3. Fill in the missing links.
